modes/config-mode/src/index.js

```diff
@@ -14,44 +14,90 @@ const ConfigurableModes = ConfigPoint.createConfiguration("ConfigurableModes", {
   // script element tag as a umd extension element.
   umdExtensions: [
    {id: '@radical/hp-extension', src: '/umd/@radical/hp-extension/index.umd.js'},
+   {id: '@radical/site-finding', src: '/umd/@radical/site-finding/index.umd.js'},
+   {id: '@radical/ecg-dicom', src: '/umd/@radical/ecg-dicom/index.umd.js'},
+  //  {id: '@radical/microscopy-dicom', src: '/umd/@radical/microscopy-dicom/index.umd.js'},
+  ],
+
+  umdLibraries: [
+    {id: 'config-point', src: '/umd/config-point/index.umd.js'},
   ],
 });
 
+
+const amd = {};
+const modules = {};
+let counter = 0;
+
+function defineGlobal(...args) {
```

Comment:
This is still a bit of a WIP, so leaving a bunch of console logs in place until the loading is further along.